src/test/java/com/placelab/utils/WebDriverSetup.java

```diff
@@ -0,0 +1,18 @@
+package com.placelab.utils;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class WebDriverSetup {
+    public static WebDriver getWebDriver(String browser){
+        if (browser.equals("Chrome")){
+            WebDriverManager.chromedriver().setup();
+            return new ChromeDriver();
+        } else {
+            WebDriverManager.firefoxdriver().setup();
+            return new FirefoxDriver();
+        }
```

Comment:
when writing methods that depend on the user input, we should design it appropriately - to always make sure that the invalid inputs are supported in a meaningful way. If I set the browser parameter to be `edge`, the test script would be run in `firefox` - and that is not what we expected. Try fixing this.